scripts/data/observations_to_datastore.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python
+
+from astropy import units as u
+from astropy.time import Time
+
+from google.cloud import datastore
+
+from pocs.base import PanBase
+from pocs.utils import current_time
+
+
+class ObservatonsExporter(PanBase):
+    """ A simple class for exporting observations to Google Datastore
+
+    Observations are pulled from the corresponding mongo collection.
+    """
+
+    def __init__(self, date=None, project_id='panoptes-survey'):
+        super(ObservatonsExporter, self).__init__()
```

Comment:
super().__init__() according to pep-3135.

https://www.python.org/dev/peps/pep-3135/

Reply:
My editor has built-in templates that pre-populate this and let me tab through the options to change and it maintains old style for compatibility. But I'll get this changed, thanks.